.github/workflows/test_common.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       #     key: venv-${{ matrix.os }}-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
 
       - name: Install dependencies
-        run: poetry install --no-interaction --with sentry-sdk
+        run: poetry install --no-interaction --with sentry-sdk,airflow
```

Comment:
please do not add this dependency here. the whole structure of test_common is such that we start with bare deps (sentry-sdk will go out soon as well) and then gradually increase the number of deps.
this is to catch any problems with people assuming certain dependencies exist while the bare library does not support them

in case of this PR we are changing the test code but still we should not affect how other tests are run